### PR TITLE
feat: added cart page

### PR DESCRIPTION
**Pull request by amirnathani9**
## change logs
- added cart page
- my cards
- price details

## screenshot
image

## preview link
https://deploy-preview-6--watchesworld.netlify.app/pages/cart.html

### CODE REVIEW

**Review by ckahub — COMMENTED** [review]
LGTM

**Review by devdeadviz — APPROVED** [review]
Looks good to me!

File: pages/cart.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                <button class="btn primary-outline-btn font-size-5 border-radius-1">Proceed to Checkout</button>
+            </div>
+        </section>
+    </div>
+</div>
+
+<script src="../script/navbar.js"></script>
+
+<body></body>
+
```
Comment (darshandamre) [inline pages/cart.html]: you have written your markup outside body tag

**Review by darshandamre — COMMENTED** [review]
there is a issue in your html

**Review by xii-vi — APPROVED** [review]
_Everything is in place. Good work on clean code structure. _

File: pages/cart.html (hunk 1 above)
Comment (amirnathani9) [reply pages/cart.html]: resolved the issue

**Pull request merged**